### DISCUSSION TITLE
how can i use  crepe for fo method？

### DISCUSSION THREAD

**Discussion by eaaajay** [voicepaw/so-vits-svc-fork#487] [Q&A] [answered]
1.if i use command "svc pre-hubert" rather than command "svc pre-hubert -fm crepe" ,can i use crepe for inference? 
2.if i use command "svc pre-hubert -fm crepe", can i use other f0 method?

**Comment by 34j** [comment, marked as answer]
Yes, probably

**Reply by eaaajay** [reply]
thanks ，then i have another question: what is the difference between command "svc pre-hubert -fm crepe" and command "svc pre-hubert"?
